client/wfnews-war/src/main/angular/src/app/components/current-alert-component/current-alert-component.component.html

```diff
@@ -0,0 +1,58 @@
+<div class="headers">
+    <div class="field">
+        <mat-label class="label">Notice Type</mat-label>
+        <select appWFSingleSelect appWFPlaceholder="Select..."
+                aria-label="Type"
+                [options]="alertTypeOptions"
+                [selected]="selectedAlertType"
+                (updated)="selectAlertUpdated($event)">
+            <option *ngFor = "let item of alertTypeOptions" value="{{item}}">
+                <span>{{item}}</span>
+            </option>
+        </select>
+    </div>
+
+    <div class="field"> 
+        <div class="toggle-bar">
+            <span>All Notices</span>
+            <mat-slide-toggle>Notices Near Me</mat-slide-toggle>
+        </div>
+    </div>
+</div>
+
+<div class="contents">
+    <div class="left-panels">
+        <mat-expansion-panel [expanded]="true">
+            <mat-expansion-panel-header>
+            <mat-panel-title>
+                Evacuation Orders and Alerts
+            </mat-panel-title>
+            </mat-expansion-panel-header>
+            <panel-evacuation-orders-and-alerts></panel-evacuation-orders-and-alerts>
```

Comment:
I don't think it makes sense to show the existing left-hand nav panels? The ones we're styling based off of PM are quite different?

Reply:
Or is that coming in a follow up PR?